tests/test_calculator.hh

```diff
@@ -570,6 +573,84 @@ namespace rascal {
     size_t max_angular{4};
   };
 
+  template <class BaseFixture, internal::RadialBasisType RadialType,
+            internal::AtomicSmearingType SmearingType,
+            internal::OptimizationType OptType>
+  struct RadialIntegralHandlerFixture : MultipleStructureFixture<BaseFixture> {
+    using Parent = MultipleStructureFixture<BaseFixture>;
+    using Manager_t = typename Parent::Manager_t;
+    using RadialIntegral_t =
+        internal::RadialContributionHandler<RadialType, SmearingType, OptType>;
+
+    RadialIntegralHandlerFixture() : Parent{} {
+      // filter out the hypers that don't correspond to the current Type
+      std::vector<json> hypers_temp;
+      // here it directly sets GTO DVR and only
```

Comment:
I don't understand this comment. Is the end of the sentence missing?

Reply:
neither do I. I guess @agoscinski should look at it.

Reply:
I reworded the block of comment